lib/Backends/Backends.cpp

```diff
@@ -62,3 +62,14 @@ Backend *glow::createBackend(BackendKind backendKind) {
   // always covers all possible values.
   llvm_unreachable("unreachable");
 }
+
+CostVal Backend::getOpCost(const Node &N) const {
+  // Gather the input types into a vector. Note that we do not need to do so for
+  // the result types because they are already contained in the node.
+  std::vector<TypeRef> inputTypeRefs;
```

Comment:
nit: but when we know size in advance, just allocate it right away in vector.

Reply:
Done